serverless.yml

```diff
@@ -34,6 +34,8 @@ plugins:
 
 provider:
   name: aws
+  role:
```

Comment:
Thinking about this more, I don't think we should do this in the default `provider` and instead should create a new function that reuses `base.handler` and have that one with the custom lambda role. Put another way, this shouldn't be the default, it should be separate and conspicuously commented in SLS and TF with `# OPTION(lambda-role)` or something (like and distinguishable from `OPTION(vpc)`, etc.)

Reply:
I think we still need a comment here for adding the role...

Reply:
Done!